src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindPaymentAccountsByEmailTest.java

```diff
@@ -241,6 +244,25 @@ public void get_request_returns_correct_payment_accounts_associated_with_email_f
 
     }
 
+    @Test
+    public void get_request_returns_404_when_unknown_email_passed_in_header_external() {
+        String userId = settingUpOrganisation(puiUserManager);
+        Map<String, Object> orgResponse = professionalReferenceDataClient
+                .findPaymentAccountsByEmailFromHeaderForExternalUsers("dummy23@dummy.com", puiUserManager, userId);
+        assertThat(orgResponse.get("http_status")).isEqualTo("404");
+        assertThat((String)orgResponse.get("response_body")).contains(ERROR_MSG_NO_ORGANISATION_FOUND);
+    }
+
+    @Test
+    public void get_request_returns_404_when_organisation_doesnt_have_pba_email_passed_in_header_external() {
+        Pair<String,String> userInfo = settingUpMinimalFieldOrganisation(puiUserManager);
+        Map<String, Object> orgResponse = professionalReferenceDataClient
+                .findPaymentAccountsByEmailFromHeaderForExternalUsers(userInfo.getFirst(), puiUserManager,
+                        userInfo.getSecond());
+        assertThat(orgResponse.get("http_status")).isEqualTo("404");
+        assertThat((String)orgResponse.get("response_body")).contains(ERROR_MSG_NO_PBA_FOUND);
+    }
+
     private void responseValidate(Map<String, Object> orgResponse) {
 
         assertThat(orgResponse.get("http_status").toString().contains("200"));

```

Comment:
Just a trivial suggestion - I think it is a better practice to use `HttpStatus.OK` instead of using the numerical code directly.

Reply:
Yes you are correct. But the structure of test cases designed is response is converted into map and all values in map are string. So need to follow this way.